correct the shift+insert behavior to paste selection

chenzhiwei opened the pull request:
The `shift+insert` behavior in other popular terminals is **paste selection**, so I submit PR to make sure deepin-terminal has same behavior as others.

This will help users easily migrate from another terminal to deepin-terminal.

avently commented:
@manateelazycat is something wrong with this PR?

BLumia commented:
@hualet I don't have privilege to merge this PR on GitHub, could you take a look? Thx.

hualet commented:
Building the infrastructure still takes some time, I've manually checked the build and the function, so I'll merge this now.

chenzhiwei committed:
fix: Correct the shift+insert behavior to paste selection

The shift+insert behavior in other popular terminals is paste selection,
so I submit PR to make sure deepin-terminal has same behavior with others.

Log: Correct the shift+insert behavior to paste selection

deepinAdmin reviewed (commented):
Patch Set 1:

(3 comments)

代码注释率

deepinAdmin reviewed (commented):
Patch Set 1:

commit check success!

deepinAdmin reviewed (commented):
Patch Set 1:

cppcheck success!

deepinAdmin reviewed (changes requested):
Patch Set 1: Code-Review-1

juteman commented:
HeYa!

Submissions about adding new features that are very difficult to merged. This is because adding new features does not align with the product manager's design.

XMuli commented:
And the terminal of Deepin series is always `ctrl + shift + c/v`, which is more in line with the habit; while `shift + insert ` feels more difficult to use (personal opinion)

chenzhiwei commented:
`shift + insert` to paste selection is a defacto standard for terminal applications.

With `shift + insert` and `ctrl + shift + v`, users can paste two code snippets at same time(paste selection and paste clipboard).

chenzhiwei commented:
@juteman  I think this is fix, not a feature.

I just correct the shortcut to a well-known behavior in other terminal applications.

juteman commented:
> @juteman I think this is fix, not a feature.
> 
> I just correct the shortcut to a well-known behavior in other terminal applications.

Well ,I can think this is fix. But product manager does not think so.

chenzhiwei commented:
@deepinAdmin Are there any new updates on this PR?

chenzhiwei commented:
Hello, Deepin 20.2 was released two days ago, and it does not include this enhancement.

Could you tell me the plan on this? If Deepin does not plan to include this enhancement in future, please let me know and I will close this PR.

chenzhiwei commented:
@ch593030323 ptal

juteman commented:
> Hello, Deepin 20.2 was released two days ago, and it does not include this enhancement.
> 
> Could you tell me the plan on this? If Deepin does not plan to include this enhancement in future, please let me know and I will close this PR.



> Hello, Deepin 20.2 was released two days ago, and it does not include this enhancement.
> 
> Could you tell me the plan on this? If Deepin does not plan to include this enhancement in future, please let me know and I will close this PR.

Sorry, I don't think this pull request will  be taked as I said last year. u can just close it.

chenzhiwei commented:
sure
